spyder/plugins/mainmenu/plugin.py

```diff
@@ -0,0 +1,251 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+Main menu Plugin.
+"""
+
+# Standard library imports
+from collections import OrderedDict
+import sys
+
+# Third party imports
+from qtpy.QtCore import Qt
+from qtpy.QtGui import QKeySequence
+
+# Local imports
+from spyder.api.exceptions import SpyderAPIError
+from spyder.api.plugins import SpyderPluginV2, SpyderDockablePlugin
+from spyder.api.translations import get_translation
+from spyder.api.widgets.menus import ApplicationMenu
+from spyder.plugins.mainmenu.api import ApplicationMenus, HelpMenuSections
+from spyder.utils.qthelpers import set_menu_icons
+
+# Localization
+_ = get_translation('spyder')
+
+
+class MainMenu(SpyderPluginV2):
+    NAME = 'mainmenu'
+    CONF_SECTION = 'mainmenu'
```

Comment:
Could use

```
    NAME = 'mainmenu'
    CONF_SECTION = NAME
``` 

To keep them in sync. Ideally the `CONF_SECTION
` class attribute should go away.